### ISSUE TITLE
Subtitles the wrong size after each startup

### ISSUE THREAD

**Issue by stephanedupont** [Stremio/stremio-bugs#422]
When launching Stremio and launching a video, the subtitles are bigger than they are supposed to be. Let's say for example that the subtitles were set on 100%. If I close Stremio and launch it again, the subtitles are now bigger. If I adjust the size by clicking + or -, they are again at the correct size. If I for example click on the +, they go from 100% to 110%, but as they were too big, the subtitles are actually reduced when going to 110%. If I now click on the -, they are reduced to the correct 100% size.

**To Reproduce**
1. Launch any video with subitltes
2. Close Stremio
3. Open Stremio again
4. Play the video again, subtitles will be too big
5. Adjust the subtitle size, they are now at the correct size

**Expected behavior**
Subtitles should be at their correct size

**Desktop:**
 - OS: Windows 11
 - Browser: Chrome
 - Version: several versions. Happens for more than one year on several computers I used.

**Comment by TRtomasz** [comment]
This happens only with embedded subtitles as far i know.

**Comment by Fritz-the-fox** [comment]
Yes I've been having that issue for as long as I use the app (macOS), when you start Stremio and watch a film with embedded subs, they appears way bigger and the it displays 100% in the setting, you have to click + or - and then it gets back to the right ratio.

**Comment by ItzTyrone** [comment]
I'm having the same issue, any updates?

**Comment by TRtomasz** [comment]
Sorry no updates, this is not a priority as it has very low impact

**Comment by Madman600** [comment]
It might not be high priority and not a severe issue but it is super annoying. Every time I start stremio I have to fix the subtitles. I don't bother using stremio any more because of it. Seems like it can't be that hard to fix. I would contend that an issue that causes a user to have to repeat some action to fix it every time they start the app is more than minor.

**Comment by fedew04** [comment]
I just wanna add that I've been having this issue for a long time too and it's pretty annoying.

**Comment by Fritz-the-fox** [comment]
Yeah it's definitely not a critical issue, BUT it is very annoying and I think we should consider: time to fix it/annoyance. And I am not dev but I am pretty sure this fix doesn't require that much work. Although it is still work and final decision goes to the great devs involve. But relating to the global quality of this app, this one must go away…

**Comment by CptSub** [comment]
As others said, it's a very annoying issue. Not only the size is not saved (that's fixed with a quick +/- on the keyboard after starting playback, so could be worse), but also subtitles color, and to fix that, you need to go to settings and change the color slightly so it saves the new color. Otherwise the subtitles are white after every app restart, which is eye-searing bright on a hdr oled tv.

**Comment by lothar-cell** [comment]
Any updates?

**Comment by TRtomasz** [comment]
Issue should be fixed in new shell version Stremio 5

**Comment by Acecool** [comment]
The bug still exists in Stremio 5 beta.

**Comment by lothar-cell** [comment]
@TRtomasz reopen the issue please?

**Comment by fedew04** [comment]
The issue is fixed for me
